controllers/volumereplication_controller.go

```diff
@@ -74,7 +74,9 @@ type VolumeReplicationReconciler struct {
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.7.0/pkg/reconcile
 func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 
-	_ = r.Log.WithValues("controller_volumereplication", req.NamespacedName)
+	prevLogger := r.Log
```

Comment:
why these changes are required?

Reply:
because of this reason https://github.com/openshift/ocs-operator/pull/990#discussion_r558330313

Moreover `VolumeReplicationReconciler` gets generated once when we start the operator and we are changing its value r.log field which will be retained even when it returns from reconciling. So it does not make sense to keep the requested value in r.log IMO.